static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -184,6 +183,11 @@ $gutter-large: 55px;
     // Optically aligns slashes
     margin-left: 2px;
     margin-right: -1px;
+
+    &:focus,
+    &:hover {
+        text-decoration: none;
```

Comment:
Decided not to currently just as it's mobile devices only, but will create a separate PR with focus states.